ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__ocp4.14-lp-interop.yaml

```diff
@@ -101,12 +101,18 @@ resources:
       memory: 200Mi
 tests:
 - as: operator-e2e-interop-aws-ocp414
-  cron: 0 6 23 11 *
+  cron: 0 12 * * 3
   remote_api: true
   steps:
     cluster_profile: aws-cspi-qe
     env:
       BASE_DOMAIN: cspilp.interop.ccitredhat.com
+      FIREWATCH_CONFIG: |
+        [
+          {"step": "ipi-*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "LPTOCPCI", "jira_epic": "LPTOCPCI-290"}
```

Comment:
We should be able to catch more steps then just ipi-*

"step": "operator-e2e*"
"step": "knative-serving*"
"step": "knative-eventing*"

Reply:
Yeah, here is a list of steps to define fore this sceanrio:
![image](https://github.com/openshift/release/assets/44474682/d16291ef-3e39-4cf7-b85f-adc355908cf0)

Reply:
@liswang89 Here is how you could set up the config. I have tested it on my local machine, and it seems to work fine. Please note that you will need to fill in the information about PQE's Jira project and the classification values where it is blank.

```json
[
    {"step": "ipi-*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "LPTOCPCI", "jira_epic": "LPTOCPCI-290"},
    {"step": "install-operators", "failure_type": "all", "classification": "Infrastructure Provisioning - Operators", "jira_project": "LPTOCPCI", "jira_epic": "LPTOCPCI-290"},
    {"step": "gather-*", "failure_type": "all", "classification": "Other", "jira_project": "LPTOCPCI", "jira_epic": "LPTOCPCI-290"},
    {"step": "openshift-*", "failure_type": "all", "classification": "Other", "jira_project": "LPTOCPCI", "jira_epic": "LPTOCPCI-290"},
    {"step": "knative-eventing-kafka-broker-e2e", "failure_type": "all", "classification": "", "jira_project": "", "jira_epic": "LPTOCPCI-290"},
    {"step": "knative-serving-eventing-e2e", "failure_type": "all", "classification": "", "jira_project": "", "jira_epic": "LPTOCPCI-290"},
    {"step": "operator-e2e", "failure_type": "all", "classification": "", "jira_project": "", "jira_epic": "LPTOCPCI-290"}
]
```

Feel free to modify it and add additional rules to split up the failure types (rather than using `all` for every value).

Reply:
The PR is updated as suggested.